index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+
+  <body>
+    <header id="header">
+      
+      <nav id="nav-bar">
+        <a href="#html" class="nav-link">HTML</a> |
+        <a href="#css" class="nav-link">CSS</a> |
+        <a href="#JS" class="nav-link">JavaScript</a> |
+        <a href="#python" class="nav-link">Python</a>
+      </nav>
+      <img  id="header-img" src="https://res.cloudinary.com/growsurf-prod/image/upload/v1582211139/production/gnysw2objzekbagrqiax.png" alt="Image">
+      <br><br><br>
+
+    </header>
+    
+    <br><br><br>
+    <div id="html">
+      
+        <p>HTML is a programming language that is widely used</p>
+
+    </div>
+    <br><br><br>
+    <video id="video" width="500" height="500" controls>
+      <source src="https://www.youtube.com/watch?v=rUWxSEwctFU&ab_channel=IanRushton" type="video/mp4">
+    </video>
+    <br><br><br>
+   
+    <div id="css">
+      <p>CSS is widely used to create style sheets with HTML</p>
+        
+    </div>
+    <br><br><br>
+    <div id="JS" class="jj">
+      <p>JS is another widely used programming language</p>
+        
+    </div>
+    <br><br><br>
+    <div id="python">
+      <p>Pyhton is a great language used in data science</p>
+        
+    </div><br><br><br>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <p>to subscripe for courses please enter your e-mail</p>
+      <input type="email" id="email" placeholder="email" name="email" required>
+      
+      <br><br>
+      <input id="submit" type="submit" name="submit" value="Submit">
+    </form>
+    
+    
+    
+
+    
+
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please remove the comment in the code as removing the comments would make the code more readable